docs/docs/tasks/label_assistant_reply.md

```diff
@@ -1,3 +1,13 @@
 # Label Assistant Reply
 
-Given the following discussion, provide labels for the final prompt.
+Given the following discussion, provide labels for the final prompt. In addition
+to the
+[Prompting Guide](https://projects.laion.ai/Open-Assistant/docs/guides/prompting),
+here are specific advices on how to correctly label the reply:
+
+- "Is the message spam?" is pretty self-explainatory.
```

Comment:
not really sure if we should say "it is self explaintory" on the docs page.

Reply:
Done, clarified on what is spam in both cases